src/components/NFTproduct/Nftproduct.tsx

```diff
@@ -147,49 +142,58 @@ const Nftproduct =  () => {
 
         {auctionNFT ? (
           <TimerContainer>
-            <Text fontFamily='Roboto' fontSize='1rem' color='#323232' margin='0 23px 0px 0px'>
+            <Text fontFamily="Roboto" fontSize="1rem" color="#323232" margin="0 23px 0px 0px">
               {t('auctionsEndIn')} :
             </Text>
-            <Timer timeStamp={auctionNFT.auction.expireTimestamp} helperString='Auction'></Timer>
+            <Timer timeStamp={auctionNFT.auction.expireTimestamp} helperString="Auction"></Timer>
           </TimerContainer>
         ) : null}
 
         <BuyCard>
           {LastBidding ? (
             <LastBiddingContainer>
-              <Text fontFamily='Roboto' FontWeight='bold' fontSize='0.875rem' color='#323232' margin='0 23px 0px 0px'>
+              <Text fontFamily="Roboto" FontWeight="bold" fontSize="0.875rem" color="#323232" margin="0 23px 0px 0px">
                 {t('lastBidding')} :
               </Text>
               <Amount amount={LastBidding}></Amount>
             </LastBiddingContainer>
           ) : (
-              <LastBiddingContainer>
-                <Text fontFamily='Roboto' FontWeight='bold' fontSize='1rem' color='#323232' margin='15px auto'>
-                  {t('noBidding')}
-                </Text>
-              </LastBiddingContainer>
-            )}
+            <LastBiddingContainer>
+              <Text fontFamily="Roboto" FontWeight="bold" fontSize="1rem" color="#323232" margin="15px auto">
+                {t('noBidding')}
+              </Text>
+            </LastBiddingContainer>
+          )}
 
           <BuyButtons>
-            <ButtonWishlist nftId={nftId} type='NFTProduct' width='70%' borderRadius='4px' fontSize='1rem' />
+            <ButtonWishlist
+              nftId={nftId}
+              type="NFTProduct"
+              width="70%"
+              borderRadius="4px"
+              fontSize="1rem"
+              disabled={expiredAuction}
+            />
             <PlaceBid>
               <button
                 onClick={() => {
                   setBidOrBuy(true)
                   setIsOpen(true)
                 }}
+                disabled={expiredAuction}
               >
                 {t('placeBid')}
               </button>
             </PlaceBid>
           </BuyButtons>
           <BuyNow>
-          <button
+            <button
+              disabled={expiredAuction}
               onClick={() => {
-
-                setValue(false,listingPrice)
+                setValue(false, listingPrice)
 
                 history.push('/marketplace/buyorbid')
```

Comment:
* note: should be handle this route in another Pr https://github.com/StartFi/startfi-interface/pull/208
I will make it